docs/user/configuration/flags.md

```diff
@@ -78,6 +79,7 @@ These flags require the `dynamic-config` feature to be enabled:
 * `-server.grpc.max-concurrent-streams` Maximum number of concurrent gRPC streams (0 = unlimited)
 * `-server.grpc.max-recv-msg-size-bytes` Maximum size in bytes for received gRPC messages
 * `-server.grpc.max-send-msg-size-bytes` Maximum size in bytes for send gRPC messages
+* `-server.grpc.in-memory-addr`: Internal address used for the agent to make in-memory gRPC connections to itself. (default `agent.internal:12346`).
```

Comment:
To be clear, this in no way actually opens a port for 12346? May want to explicitly state that.